tests/foreman/cli/test_activationkey.py

```diff
@@ -1698,7 +1698,7 @@ def test_negative_update_auto_attach(self):
         except CLIFactoryError as err:
             self.fail(err)
 
-        self.assertIn("'--auto-attach': value must be one of", result.stderr)
+        self.assertIn(u"'--auto-attach': value must be one of", result.stderr)
```

Comment:
how about writing the string in a form:
```python
'\'--auto-attach\': value must be one of'
```
to follow our string-in-the-single-quotes rule?